nominees/Women’s issues

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Women’s issues within political party platforms",
+  "description": "A data story about how often women's issues appeared in the text within political party platforms",
+  "website": "https://pudding.cool/projects/votes-for-women/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/the-pudding/votes-for-women/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 5,
+      "evidenceText": "This project explores every political party platform between 1840 and 2016 to find how often women's issues appeared in the text. 45 documents. 934,068 total words. 28,854 words related to women.",
+      "evidenceURL": "https://pudding.cool/projects/votes-for-women/, https://github.com/the-pudding/votes-for-women"
+    }
+  ],
+  "sectors": [
+    "Gender and Minority Groups"
+  ],
+  "type": [
+    "software",
+    "data"
```

Comment:
@Rolikasi the [MIT license](https://github.com/the-pudding/votes-for-women/blob/master/LICENSE) indicated only caters for the software category. Unless you have a valid license from the [OSI data licenses](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md) list for the data category, I would recommend to remove the data category for now. Thanks.

Reply:
@nathanbaleeta Thanks, resolved)